### DISCUSSION TITLE
new mirror request

### DISCUSSION THREAD

**Discussion by laurentiuhosterion** [linuxmint/discussions#261] [Questions]
Hello, does anybody know how to submit a new mirror request for linuxmint?

**Comment by rcalixte** [comment]
If you are interested in hosting a new mirror, the instructions are on the [Mirrors](https://linuxmint.com/mirrors.php) page.

![2023-12-21_08-20](https://github.com/linuxmint/discussions/assets/10281587/1180d7a1-baa0-4ab4-825d-0c1dc5bf9cc0)

**Comment by laurentiuhosterion** [comment]
Yeah, I've emailed them but no answer. I will wait, maybe they are busy with current release.
